### PR TITLE
Cherrypicks

### PR DESCRIPTION
**Pull request by guineveresaenger**
Partial attempt to improve contributor experience on cherrypicks; this updates/clarifies some information in cherrypick instructions.

/sig contributor-experience
/sig release
/cc @tpepper @justaugustus @cblecker @spiffxp 

reference for tracking work on automation: https://github.com/kubernetes/test-infra/issues/1795

### CODE REVIEW

File: contributors/devel/cherry-picks.md [hunk 1]
```diff
@@ -25,17 +25,19 @@ task is for backporting PRs from master to release branches.
    `upstream/release-3.14`: `hack/cherry_pick_pull.sh upstream/release-3.14
    98765`
  * Your cherrypick PR targeted to the release branch will immediately get the
-   `do-not-merge/cherry-pick-not-approved` label. The release branch owner
+   `do-not-merge/cherry-pick-not-approved` label. The Patch Release Manager
```
Comment (spiffxp) [inline contributors/devel/cherry-picks.md]: nit: link Patch Release Manager to https://github.com/kubernetes/sig-release/blob/master/release-team/README.md#patch-release-manager

File: contributors/devel/cherry-picks.md [hunk 2]
```diff
@@ -25,17 +25,19 @@ task is for backporting PRs from master to release branches.
...
    will triage PRs targeted to the branch.  Normal rules apply for code merge.
    * Reviewers `/lgtm` and owners `/approve` as they deem appropriate.
-   * The approving release branch owner is responsible for applying the
-     `cherrypick-approved` label.
+   * Milestone approval needs to be added for the target release branch.
```
Comment (spiffxp) [inline contributors/devel/cherry-picks.md]: What does this mean?  Someone needs to apply `status/approved-for-milestone`? Who would that be?

File: contributors/devel/cherry-picks.md [hunk 3]
```diff
@@ -25,17 +25,19 @@ task is for backporting PRs from master to release branches.
...
+   * The Patch Release Manager for the target release branch is responsible for
+   applying the `cherrypick-approved` label. You can find the release team contacts
```
Comment (spiffxp) [inline contributors/devel/cherry-picks.md]: IMO you can drop "You can find the release team contacts..." if you link Patch Release Manager directly.  That description includes a link to all release / patch release branch managers

File: contributors/devel/cherry-picks.md (hunk 2 above)
Comment (tpepper) [reply contributors/devel/cherry-picks.md]: Is it correct as follows?

```
* Milestone label must be applied to match the target release branch
* SIG leads associated with the PR must indicate `/status approved-for-milestone`
```

Or does the milestone labeling happen via the cherry pick script?

File: contributors/devel/cherry-picks.md (hunk 3 above)
Comment (tpepper) [reply contributors/devel/cherry-picks.md]: Might still want a few words around how the person doing the role changes in each release and to look at the release vX.Y specific table to find the right person.

File: contributors/devel/cherry-picks.md [hunk 4]
```diff
@@ -25,17 +25,19 @@ task is for backporting PRs from master to release branches.
...
+   under the kubernetes/sig release repository - for example, [here is the 1.11 team](https://git.k8s.io/sig-release/releases/release-1.11/release_team.md).
 
 ## Cherry Pick Review
 
 Cherry pick pull requests are reviewed differently than normal pull requests. In
-particular, they may be self-merged by the release branch owner without fanfare,
-in the case the release branch owner knows the cherry pick was already
+particular, they may be self-merged by the Patch Release Manager without fanfare,
+in the case the Patch Release Manager knows the cherry pick was already
 requested - this should not be the norm, but it may happen.
```
Comment (neolit123) [inline contributors/devel/cherry-picks.md]: this section is from 2015 and arguably needs a 2018 refactor outlining the current "norm".
i can summarize, if needed.

Comment (guineveresaenger) [reply contributors/devel/cherry-picks.md]: @neolit123 yes, please! I'm not super familiar with the process; I stumbled across it because of a coworker's struggles.

File: contributors/devel/cherry-picks.md (hunk 1 above)
Comment (guineveresaenger) [reply contributors/devel/cherry-picks.md]: ahhhhh yes this is what I was looking for. Thank you!

File: contributors/devel/cherry-picks.md (hunk 3 above)
Comment (guineveresaenger) [reply contributors/devel/cherry-picks.md]: @tpepper - thanks to @spiffxp 's link in comment above, I think the flow of information is relatively clear and obvious to infer.

File: contributors/devel/cherry-picks.md (hunk 2 above)
Comment (guineveresaenger) [reply contributors/devel/cherry-picks.md]: AFAIK, they do not. The two sample cherry-picks I saw did not have them automatically included; they were added manually by the reviewer.
I was going for something like:

Normal rules apply for code merge, such as:
- reviews and lgtms
- milestones and approvals
- passing tests
- hearts and smilies

and then focus on what's *different* in cherry picks.

**Comment by guineveresaenger** [comment]
@spiffxp @tpepper @neolit123 - PTAL

**Comment by guineveresaenger** [comment]
@neolit123 I'm not entirely sure where to put that information, or if it even belongs here.

Your first two points are true of any PR, if there's conflicts or failing tests it is up to the contributor to fix those. I think it's clear that cherry-pick PRs are no different.
To your last point, if a member is following the recommended github flow, the cherry pick script should run as avertised, but I can see the value of a quick reminder to call one's upstream `upstream`.

**Comment by neolit123** [comment]
@guineveresaenger good point.

File: contributors/devel/cherry-picks.md (hunk 2 above)
Comment (tpepper) [reply contributors/devel/cherry-picks.md]: So I feel like this is ok then?  It says "Normal rules apply for code merge" and milestone approval requirements are maybe hopefully sort of going to appear in contributors/devel/issues.md in my https://github.com/kubernetes/community/pull/2430?  But now that I say that here and have posted small updates there, I wonder if that file should not be issues.md but rather something like milestones.md since it's trying to cover issues and PRs?

File: contributors/devel/cherry-picks.md [hunk 5]
```diff
@@ -25,18 +25,20 @@ task is for backporting PRs from master to release branches.
    `upstream/release-3.14`: `hack/cherry_pick_pull.sh upstream/release-3.14
    98765`
  * Your cherrypick PR targeted to the release branch will immediately get the
-   `do-not-merge/cherry-pick-not-approved` label. The release branch owner
+   `do-not-merge/cherry-pick-not-approved` label. The [Patch Release Manager](https://git.k8s.io/sig-release/release-team/README.md#patch-release-manager)
```
Comment (nikhita) [inline contributors/devel/cherry-picks.md]: :+1:

File: contributors/devel/cherry-picks.md (hunk 2 above)
Comment (nikhita) [reply contributors/devel/cherry-picks.md]: > SIG leads associated with the PR must indicate `/status approved-for-milestone`

This is only required during code slush. Otherwise, this label is not really used: https://github.com/kubernetes/kubernetes/pulls?q=is%3Apr+label%3Acherrypick-approved+is%3Aclosed.

Comment (nikhita) [reply contributors/devel/cherry-picks.md]: Follow up question: when is `cherrypick-candidate` used and how is it different from `cherrypick-approved`? There does not seem to be any consistency in how this label is being used right now.

File: contributors/devel/cherry-picks.md [hunk 6]
```diff
@@ -25,18 +25,20 @@ task is for backporting PRs from master to release branches.
...
+   applying the `cherrypick-approved` label.
 
 ## Cherry Pick Review
 
-Cherry pick pull requests are reviewed differently than normal pull requests. In
-particular, they may be self-merged by the release branch owner without fanfare,
-in the case the release branch owner knows the cherry pick was already
-requested - this should not be the norm, but it may happen.
+Cherry pick pull requests have an additional requirement compared to normal pull
+requests. They must be approved specifically for cherry-pick by SIG leads.
```
Comment (nikhita) [inline contributors/devel/cherry-picks.md]: SIG leads' approval is not really required, unless during code slush where the SIG lead will need to apply the `status/approved-for-milestone` label.

But in general, anyone who has the power to `/approve` the relevant changes in the PR can approve the cherry-pick.

**Comment by nikhita** [comment]
Also, some good info we can add here would be about cherry-picking to multiple release branches. Ref: https://github.com/kubernetes/community/issues/994.

**Comment by guineveresaenger** [comment]
@nikhita I'm not sure I entirely understand. I'm assuming you'd run the script with different aruments in that case, as you'd have had to make changes in case of a merge conflict or failing test.

**Comment by guineveresaenger** [comment]
@neolit123 @nikhita - PTAL
I tried to address the most common concerns and questions. If you could look at it in the light of my questions above to make sure I captured the correct state of things. Thank you!

File: contributors/devel/cherry-picks.md [hunk 7]
```diff
@@ -19,26 +20,55 @@ task is for backporting PRs from master to release branches.
...
    This example applies a master branch PR #98765 to the remote branch
    `upstream/release-3.14`: `hack/cherry_pick_pull.sh upstream/release-3.14
    98765`
+   * Be aware the cherrypick script assumes you have a git remote called `upstream` that points at the Kubernetes github org.
+   It will also expect the following environment variables:
+   ```bash
+   FORK_REMOTE=<remote-of-your-fork> \
+   UPSTREAM_REMOTE=<remote-of-upstream> \
+   GITHUB_USER=<your-github-user> \
+   ./hack/cherry_pick_pull.sh <remote-of-upstream>/<release-x.xx> <master-branch-pr-number>
```
Comment (nikhita) [inline contributors/devel/cherry-picks.md]: I'm not sure if we should include all this info here. 

> ./hack/cherry_pick_pull.sh <remote-of-upstream>/<release-x.xx> <master-branch-pr-number>

This is not needed because an example already exists above.

> GITHUB_USER=<your-github-user>

This is useful because this is _really needed_. The script won't work without it. But if you don't set this env var, the script will prompt you to do that. We can include this. :+1: 

```
FORK_REMOTE=<remote-of-your-fork> \
   UPSTREAM_REMOTE=<remote-of-upstream> \
```

Looking at this, it reads to me as I _need_ to set these env vars to get the script to work. This is not true because the script will assume the default fork as "origin" and upstream as "upstream". If someone doesn't have the remotes set this way, only then would they have to set these variables.

Ref: the script has these lines:

```
UPSTREAM_REMOTE=${UPSTREAM_REMOTE:-upstream}
FORK_REMOTE=${FORK_REMOTE:-origin}
```

@neolit123 @guineveresaenger thoughts?

File: contributors/devel/cherry-picks.md [hunk 8]
```diff
@@ -19,26 +20,55 @@ task is for backporting PRs from master to release branches.
...
 
 See the [cherrypick queue dashboard](http://cherrypick.k8s.io/#/queue) for
 status of PRs labeled as `cherrypick-candidate`.
+
+## Troubleshooting Cherrypicks
+
+Contributors may encounter some of the following difficulties when initiating a cherrypick.
+
+- A cherrypick PR does not apply cleanly against an old release branch.
+In that case, you will need to manually fix conflicts and rebase.
```
Comment (nikhita) [inline contributors/devel/cherry-picks.md]: Do we need the `and rebase` in the end? I think fixing conflicts is enough. If we rebase, what do we rebase on top of?

File: contributors/devel/cherry-picks.md [hunk 9]
```diff
@@ -19,26 +20,55 @@ task is for backporting PRs from master to release branches.
...
+
+- The cherrypick PR includes code that does not pass CI tests.
+In such a case you will have to fetch the auto-generated branch from your fork, amend the problematic commit and force push to the auto-generated branch.
+Alternatively, you can create a new PR, which is noisier.
+
+- In the case of a rebase or cherrypick specific fix, you can run the cherrypick script again and it will automatically force-push to the autogenerated cherrypick branch.
```
Comment (nikhita) [inline contributors/devel/cherry-picks.md]: @guineveresaenger my bad, I don't think should be included here. :)

What I really meant:
This is can only occur if you created a cherry-pick PR before the parent PR has merged. Then something changed in the parent PR. This change should be reflected in the cherry-pick PR. To do so, you can run the script again, which will pick the changes from the parent PR and force push to the autogenerated branch.

But this is not the ideal way the script should be used, so we should probably not include it here. :)

File: contributors/devel/cherry-picks.md (hunk 6 above)
Comment (nikhita) [reply contributors/devel/cherry-picks.md]: > it's not that it takes a SIG lead; approvers have the power of adding the cherry-pick approval. Yes?

Sorry! I should have been clearer.

> Cherry pick pull requests have an additional requirement compared to normal pull
requests. They must be approved specifically for cherry-pick by SIG leads.

I meant that cherry-picks don't need any approval from SIG leads.

- To get the `approved` label, anyone within the approvers list of the relevant OWNERS file can `/approve`.
- To get the `cherrypick-approved` label, only the patch release manager can add the label.

File: contributors/devel/cherry-picks.md [hunk 10]
```diff
@@ -1,12 +1,13 @@
 # Overview
 
-This document explains how cherry picks are managed on release
-branches within the Kubernetes projects. A common use case for this
+This document explains how cherrypicks are managed on release
+branches within the Kubernetes projects.
```
Comment (nikhita) [inline contributors/devel/cherry-picks.md]: Isn't this only for k/k i.e. not all Kubernetes projects?

File: contributors/devel/cherry-picks.md [hunk 11]
```diff
@@ -19,26 +20,55 @@ task is for backporting PRs from master to release branches.
...
    98765`
+   * Be aware the cherrypick script assumes you have a git remote called `upstream` that points at the Kubernetes github org.
+   It will also expect the following environment variables:
+   ```bash
+   FORK_REMOTE=<remote-of-your-fork> \
+   UPSTREAM_REMOTE=<remote-of-upstream> \
+   GITHUB_USER=<your-github-user> \
+   ./hack/cherry_pick_pull.sh <remote-of-upstream>/<release-x.xx> <master-branch-pr-number>
+   ```
+   * You will need to run the cherrypick script separately for each patch release you want to cherrypick to.
```
Comment (nikhita) [inline contributors/devel/cherry-picks.md]: :heart:

**Comment by nikhita** [comment]
> When we say "oldest branch" does that mean "1.12 is older than 1.11?" This is unclear to me - in terms of existence, 1.12 is older, in terms of creation date, 1.11 is. But maybe this is common parlance that I'm unaware of.

Oh, I see that you mean this from the comment here: https://github.com/kubernetes/community/issues/994#issuecomment-325790382.

I think the comment meant that the parent PR should contain the milestone for the oldest branch (datewise) i.e. if a parent PR contains cherry-pick to 1.11 and 1.10, it should get the 1.10 milestone.

But I'm not sure if that's correct? It is technically a PR against master (1.12 in the current case) and only the cherry-pick PRs are against the older branches (so they should get 1.11 and 1.10 milestones).

@tpepper thoughts?

File: contributors/devel/cherry-picks.md (hunk 7 above)
Comment (neolit123) [reply contributors/devel/cherry-picks.md]: > UPSTREAM_REMOTE=${UPSTREAM_REMOTE:-upstream}
FORK_REMOTE=${FORK_REMOTE:-origin}

it depends on the git workflow, i guess.

1.

the steps here suggest cloning upstream:
https://github.com/kubernetes/kubernetes/blob/master/README.md#to-start-developing-kubernetes
`git clone https://github.com/kubernetes/kubernetes`

this will set `origin` being the default remote for upstream `k/k`.

from there users can add their fork as another remote (this is what i do).
so with this workflow the `cherry_pick_pull.sh` defaults are off and the users need to rename remotes or use the env. variables.

2.

this is the official workflow however:
https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md#2-clone-fork-to-local-storage
it suggests to do:
`git clone https://github.com/$user/kubernetes.git`
and them setup`upstream`, which will be compliant with the `cherry_pick_pull.sh` defaults.

the github suggested workflow is the same:
https://help.github.com/articles/configuring-a-remote-for-a-fork/

so if assume that the users always follow 2. there is no need to set the env.vars.

File: contributors/devel/cherry-picks.md (hunk 9 above)
Comment (neolit123) [reply contributors/devel/cherry-picks.md]: @nikhita @guineveresaenger 
i think it was added because i made a comment somewhere in this discussion.
but please feel free to remove / improve it if it feels like an edge case.

there is also the case where you create the cherry pick PR using the script, but then for instance CI tests do not pass or something about the code in the old branch is different, and while the diff applies, changes only to the cherry pick PR have to be made.

File: contributors/devel/cherry-picks.md (hunk 10 above)
Comment (guineveresaenger) [reply contributors/devel/cherry-picks.md]: hah, I think that's someone's typo (or otherwise optimism re: other k/repos) and yes. good catch!

**Comment by guineveresaenger** [comment]
@nikhita - so, they should get the milestone of the minor version they're being cherrypicked into? that seems intuitive but not what I initially thought it said.

File: contributors/devel/cherry-picks.md (hunk 9 above)
Comment (guineveresaenger) [reply contributors/devel/cherry-picks.md]: yeah those are really special circumstances - I'll take it out.

File: contributors/devel/cherry-picks.md (hunk 8 above)
Comment (guineveresaenger) [reply contributors/devel/cherry-picks.md]: Yep - too much detail can get confusing. Will take out.

File: contributors/devel/cherry-picks.md (hunk 7 above)
Comment (guineveresaenger) [reply contributors/devel/cherry-picks.md]: whoa, I had _no idea_ there were instructions floating around that effectively set k/k as origin. We should fix that.

Comment (neolit123) [reply contributors/devel/cherry-picks.md]: `origin` as the upstream remote predates github. :)

**Comment by nikhita** [comment]
> I'm pretty sure all milestones for the original cherrypick candidate I've seen have been for the latest milestone.

:+1:

I think the issue just made us all confused. :)

**Comment by nikhita** [comment]
This lgtm from my side once the commits are squashed and https://github.com/kubernetes/community/pull/2408#discussion_r204595469 is discussed and adressed. 

Thanks for improving this doc, @guineveresaenger! :1st_place_medal:

File: contributors/devel/cherry-picks.md (hunk 2 above)
Comment (guineveresaenger) [reply contributors/devel/cherry-picks.md]: I assumed `cherrypick-candidate` was a nonnecessary pre-stage for `cherrypick-approved`?

or perhaps it is related to the release cycle?

**Comment by guineveresaenger** [comment]
This is my current best notion of the conditions surrounding the  [discussion on #2408 (comment) ](https://github.com/kubernetes/community/pull/2408#discussion_r204595469).

Milestone requirements seem in need of becoming a bit more rigorous and it looks like they're headed that way, but as they're currently not required I went with an "if you're going to have one, make it match the target release branch".

Do we have an issue to clarify the process surrounding whether something's in a milestone? @tpepper? @justaugustus ?

**Comment by tpepper** [comment]
I don't see any open issues in k/features or k/sig-release around milestone requirements.

There is https://github.com/kubernetes/release/issues/571 on the cherry-pick specific side of the house.

If SIG-PM brings forward a process change proposal KEP on feature management, I expect a lot of discussion around it.  Right now there's very early draft discussing happening in some google docs. Haven't heard a timeline for solidifying a proposal.

So today something is:
 * "in a milestone" (aka "in a release") if it merges to master ahead of release, or is cherry picked
 * "in a Milestone" (aka annotated with GitHub Milestone "vX.YY" metadata) if a https://github.com/orgs/kubernetes/teams/kubernetes-milestone-maintainers member has marked it as such manually or via prow "/milestone"

We have an overloaded term.

**Comment by tpepper** [comment]
I think this is going to need sync'd up with https://github.com/kubernetes/test-infra/pull/9128

File: contributors/devel/cherry-picks.md [hunk 12]
```diff
@@ -19,26 +19,45 @@ task is for backporting PRs from master to release branches.
...
+
  * Your cherrypick PR targeted to the release branch will immediately get the
-   `do-not-merge/cherry-pick-not-approved` label. The release branch owner
+   `do-not-merge/cherry-pick-not-approved` label. The [Patch Release Manager](https://git.k8s.io/sig-release/release-team/README.md#patch-release-manager)
    will triage PRs targeted to the branch.  Normal rules apply for code merge.
    * Reviewers `/lgtm` and owners `/approve` as they deem appropriate.
-   * The approving release branch owner is responsible for applying the
-     `cherrypick-approved` label.
+   * Milestones on cherrypick PRs should be the milestone for the target release branch (e.g.milestone 1.11 for a cherrypick onto release-1.11).
```
Comment (justaugustus) [inline contributors/devel/cherry-picks.md]: s/`e.g.milestone`/`e.g., milestone`

File: contributors/devel/cherry-picks.md [hunk 13]
```diff
@@ -19,26 +19,45 @@ task is for backporting PRs from master to release branches.
...
+requests.
+They must be approved specifically for cherrypick by Approvers.
+Additionally, the [Patch Release Manager](https://git.k8s.io/sig-release/release-team/README.md#patch-release-manager)
+is the final and sole authority on removing the `do-not-merge/cherry-pick-not-approved`
+label and triggering a merge into the current patch release.
 
-## Searching for Cherry Picks
+## Searching for Cherrypicks
 
 See the [cherrypick queue dashboard](http://cherrypick.k8s.io/#/queue) for
```
Comment (justaugustus) [inline contributors/devel/cherry-picks.md]: This dashboard doesn't load now, probably because the `cherrypick-candidate` label [no longer exists](https://github.com/kubernetes/test-infra/pull/9091). Filed this to fix / remove: https://github.com/kubernetes/k8s.io/issues/140

**Comment by guineveresaenger** [comment]
Thank you @justaugustus! I’m behind due to new environment setup and CLA Transfers but thank you so much for the review. Will follow up ASAP.

File: contributors/devel/cherry-picks.md (hunk 12 above)
Comment (guineveresaenger) [reply contributors/devel/cherry-picks.md]: regarding your first question, the patch release manager applies the `cherrypick-approved` label.

Comment (guineveresaenger) [reply contributors/devel/cherry-picks.md]: oh I see your question now - sorry! 😓 I'm honestly not sure, will ask.

**Comment by guineveresaenger** [comment]
@justaugustus - I swear, it used to say "cherrypick"! I triple checked!

File: contributors/devel/cherry-picks.md (hunk 12 above)
Comment (nikhita) [reply contributors/devel/cherry-picks.md]: > AFAIK the patch release manager applies the cherrypick-approved label.

Yes, this is correct. :)

The patch release manager is responsible for approving cherrypicks.

File: contributors/devel/cherry-picks.md (hunk 2 above)
Comment (guineveresaenger) [reply contributors/devel/cherry-picks.md]: I looked for labels in k/k and cherrypick-candidate is obsolete. Out it goes.

File: contributors/devel/cherry-picks.md [hunk 14]
```diff
@@ -19,26 +19,46 @@ task is for backporting PRs from master to release branches.
...
+Contributors may encounter some of the following difficulties when initiating a cherry-pick.
 
-## Searching for Cherry Picks
+- A cherry-pick PR does not apply cleanly against an old release branch.
+In that case, you will need to manually fix conflicts.
 
-See the [cherrypick queue dashboard](http://cherrypick.k8s.io/#/queue) for
-status of PRs labeled as `cherrypick-candidate`.
+- The cherry-pick PR includes code that does not pass CI tests.
+In such a case you will have to fetch the auto-generated branch from your fork, amend the problematic commit and force push to the auto-generated branch.
```
Comment (spiffxp) [inline contributors/devel/cherry-picks.md]: You have this statement on a new line, but markdown just joins it to the previous bullet.  Which looks fine to me, just wondering if that was your intent

**Comment by spiffxp** [comment]
/approve
/hold
I'm totally fine with the content as-is.  It'd be nice if this got word-wrapped to 80 chars.  If any of the previous reviewers are fine with this as-is, feel free to `/lgtm` and `/hold cancel`

File: contributors/devel/cherry-picks.md [hunk 15]
```diff
@@ -19,26 +19,46 @@ task is for backporting PRs from master to release branches.
...
+requests.
+They must be approved specifically for cherry-pick by Approvers.
+Additionally, the [Patch Release Manager](https://git.k8s.io/sig-release/release-team/README.md#patch-release-manager)
+is the final and sole authority on removing the `do-not-merge/cherry-pick-not-approved`
+label and triggering a merge into the current patch release.
+
+## Searching for Cherry-picks
+
+[A sample search on kubernetes/kubernetes pull requests that are labeled as `cherry-pick-approved`](https://git.k8s.io/kubernetes/pulls?q=is%3Aopen+is%3Apr+label%3Acherry-pick-approved)
+[A sample search on kubernetes/kubernetes pull requests that are labeled as `do-not-merge/cherry-pick-not-approved`](https://github.com/kubernetes/kubernetes/pulls?q=is%3Aopen+is%3Apr+label%3Ado-not-merge%2Fcherry-pick-not-approved)
```
Comment (nikhita) [inline contributors/devel/cherry-picks.md]: I think this needs a bullet point.

**Comment by nikhita** [comment]
A couple of nits, but lgtm from my side. :+1:

File: contributors/devel/cherry-picks.md (hunk 15 above)
Comment (justaugustus) [reply contributors/devel/cherry-picks.md]: Yep. This needs bullet points.
Also, the first query should be: `https://github.com/kubernetes/kubernetes/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+label%3Acherry-pick-approved`

**Comment by guineveresaenger** [comment]
@nikhita -PTAL

File: contributors/devel/cherry-picks.md (hunk 14 above)
Comment (guineveresaenger) [reply contributors/devel/cherry-picks.md]: Really!? That's super strange actually I do not see thatat all!?

**Comment by guineveresaenger** [comment]
actually, hold a sec, wanted to get some clarification on branch mgr vs. patch release mgr
